feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/OrchestratorActivity.kt

```diff
@@ -63,6 +65,13 @@ internal class OrchestratorActivity : BaseActivity() {
                 // Some co-sync functionality depends on the exact package name of the caller app,
                 // e.g. to switch content providers of debug and release variants of the caller app
                 extras.putString(ClientApiConstants.CALLER_PACKAGE_NAME, callingPackage)
+                PreferenceManager.getDefaultSharedPreferences(this)
```

Comment:
Would it be possible to use a custom shared prefs instead of the managed ones?

Reply:
Of course it is but what's the purpose for a single value?

Reply:
We could avoid the UI dependency from creeping into infra modules.